lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -46,6 +47,8 @@ import Data.Text
 import Data.Word
     ( Word16, Word8 )
 
+import GHC.Generics
```

Comment:
Avoid implicit imports, this ages badly and makes it hard to figure out where variable comes from for future reader. Prefer explicit imports :+1:

Reply:
You can use haskell-language-server to figure out where variables come from. This rule is now obsolete. But I'll comply regardless.